tests/query_resource_selector_test.go

```diff
@@ -282,3 +286,102 @@ var _ = ginkgo.Describe("Resoure Selector limits", ginkgo.Ordered, func() {
 		}
 	})
 })
+
+var _ = ginkgo.Describe("Resoure Selector with PEG", ginkgo.Ordered, func() {
+	ginkgo.BeforeAll(func() {
+		_ = query.SyncConfigCache(DefaultContext)
+	})
+
+	// = , != , item in list, item not in list, prefix, suffix, date operations (created_at, updated_at), agent query
+	testData := []struct {
+		description string
+		query       string
+		expectedIDs []uuid.UUID
+		resource    string
+	}{
+		{
+			description: "config item query",
+			query:       `name="node-b" type="Kubernetes::Node"`,
```

Comment:
```suggestion
			query:       `name=node-b type=Kubernetes::Node`,
```

Reply:
can we also add a `name=node-b type=kubernetes::node` (case insensitivity)

Reply:
Doing the suggested will cause the test to fail as things get parsed incorrectly